scripts/pds-ingress-client.py

```diff
@@ -3,11 +3,12 @@
 import argparse
 import json
 import logging
-import os
 import requests
 import subprocess
 
 from src.pds.ingress.util.config_util import ConfigUtil
+from src.pds.ingress.util.node_util import NodeUtil
```

Comment:
shoud `src.` be in the package name ? That does not sound right to me but I would not be able to tell you how to avoid that.

Reply:
I agree that this is not typical, but I think its a side effect of how the template python repo expects the source code to be organized. Would be good to get @nutjob4life to weigh in and confirm whether its necessary to include `src.` or not.

Reply:
The `setup.cfg` file says that `package_dir` is `src`, so `src` should _not_ appear in import statements.